pyccel/codegen/printing/ccode.py

```diff
@@ -488,36 +486,56 @@ def _print_PythonLen(self, expr):
                     PYCCEL_RESTRICTION_TODO,
                     symbol = expr, severity='fatal')
 
+    def _print_Header(self, expr):
+        return ''
```

Comment:
if the _print_Header does nothing can we remove it?

Reply:
No, there are headers in the code. They don't need to be printed in C, but if there is no `_print_Header` function then we will get errors saying that the printer doesn't know how to print this object. I'm not sure why they still exist after the semantic stage. If we don't need them then we could return `EmptyNode()` from the `_print_XHeader` functions in the semantic stage, but I think checking if these are used anywhere or not is beyond the scope of this PR. I'm not sure why we didn't have this problem before. We must have had it at some point as there is already a `_print_Header` function in `fcode.py`

Reply:
I think because we are using `self._print(expr)` instead of calling directly the needed print.
anyway, this is beyond this PR, we will look at that after.
we will need to create an issue for clearing things (include unused declared variables).

Reply:
> I think because we are using self._print(expr) instead of calling directly the needed print.

Yes, this is necessary to avoid an enormous number of if blocks

> anyway, this is beyond this PR, we will look at that after.

:+1:

> include unused declared variables

I'm not sure we want to do that given that we always try to produce code as close as possible to the user's code, and that the aim of this PR (and the related issues #740, #741, #742 ) is to allow modules such as:
```
g = 9.81
e = 3.14
```